### PR TITLE
builder/relay REST api

### PR DESCRIPTION
**Pull request by metachris**
Superseded by this PR: https://github.com/protolambda/mergemock/pull/26

### CODE REVIEW

File: types/builder_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
...0000000000","fee_recipient":"0x0200000000000000000000000000000000000000","state_root":[3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"receipts_root":[4,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"logs_bloom":"0x05000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","prev_randao":"0x0600000000000000000000000000000000000000000000000000000000000000","block_number":"5001","gas_limit":"5002","gas_used":"5003","timestamp":"5004","extra_data":"0x0700000000000000000000000000000000000000000000000000000000000000","base_fee_per_gas":"8","block_hash":"0x0900000000000000000000000000000000000000000000000000000000000000","transactions_root":[10,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}`
```
Comment (lightclient) [inline types/builder_test.go:33]: `transactions_root` is in the form `[x, x, ..]`, seems like it should be hex-encoded?

**Review by lightclient — COMMENTED** [review]
Looking really good so far.

File: types/builder_test.go (hunk 1 above)
Comment (metachris) [reply types/builder_test.go:33]: Good catch! It's missing the marshalling functions, adding now.

Comment (metachris) [reply types/builder_test.go]: Updated

**Comment by metachris** [comment]
Closing because superseded by https://github.com/protolambda/mergemock/pull/26

**Pull request closed without merging**